packages/shared/lib/clients/sync.client.ts

```diff
@@ -289,6 +301,8 @@ class SyncClient {
                     syncData: JSON.stringify(syncData)
                 }
             });
+            // await logCtx.error('Failed to init sync', {error: err});
+            // await logCtx.failed();
```

Comment:
commented?

Reply:
Yeah, I didn't changed broken logic but added the comment to not forget. This one of the places where the activity logs were not updated in the catch